src/lib/libidbstore.js

```diff
@@ -112,36 +112,43 @@ var LibraryIDBStore = {
   emscripten_idb_store__async: true,
   emscripten_idb_store: (db, id, ptr, num, perror) => Asyncify.handleSleep((wakeUp) => {
     IDBStore.setFile(UTF8ToString(db), UTF8ToString(id), new Uint8Array(HEAPU8.subarray(ptr, ptr+num)), (error) => {
+      /** @suppress{checkTypes} */
       {{{ makeSetValue('perror', 0, '!!error', 'i32') }}};
       wakeUp();
     });
   }),
   emscripten_idb_delete__async: true,
   emscripten_idb_delete: (db, id, perror) => Asyncify.handleSleep((wakeUp) => {
     IDBStore.deleteFile(UTF8ToString(db), UTF8ToString(id), (error) => {
+      /** @suppress{checkTypes} */
```

Comment:
These are needed because closure doesn't like storing `booleans` in TypedArrays.